monasca/templates/grafana-deployment.yaml

```diff
@@ -30,16 +30,29 @@ spec:
           - name: grafana-config
             mountPath: /etc/grafana
           env:
+            - name: GF_AUTH_KEYSTONE_AUTH_URL
+            {{- if .Values.grafana.keystone.url }}
+{{- include "monasca_secret_env" .Values.grafana.keystone.url | indent 14 }}
+            {{- else }}
+              value: "http://{{ template "keystone.fullname" . }}:{{ .Values.keystone.service.admin_port }}"
+            {{- end }}
+{{ include "monasca_keystone_env" .Values.grafana.keystone | indent 12 }}
```

Comment:
this isnt usually defined. What if we dont set values.grafana.keystone as it is now. Also what do you need from here? Dont you just need the keystone endpoitn for grafana